src/components/ScholarshipForm.jsx

```diff
@@ -41,29 +43,28 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function ScholarshipForm({ scholarship, submitFn, onSubmitError }) {
+function ScholarshipForm({ scholarship, onSubmit }) {
   const classes = useStyles();
   const [activeStep, setActiveStep] = useState(0);
-  const [noReqsChecked, setNoReqsChecked] = useState(false);
+  const [submissionError, setSubmissionError] = useState(null);
 
   const formik = useFormik({
     initialValues: scholarship.data,
     validationSchema,
     validateOnChange: false,
-    onSubmit: (values, { setSubmitting, resetForm }) => {
+    onSubmit: (values, { setSubmitting }) => {
       setSubmitting(true);
       scholarship.data = { ...values };
       scholarship
         .save()
-        .then(submitFn)
-        .then(() => setActiveStep(0))
-        .then(() => setNoReqsChecked(false))
-        .then(resetForm)
-        .catch(onSubmitError)
+        .then(onSubmit)
```

Comment:
right now after submission / edit, it doesn't reset the `ActiveStep`. 
If you spam the submit button after adding a new scholarship, it will make duplicates. 

Since we will fix this later with a dialog, for now I think we can disable the buttons, forcing them to either view the scholarship or go to another page. OR just take them back to the beginning of the form.

Reply:
Oh weird, I thought changing the `scholarship` prop of the form would prompt a recreation of this component. I'll have it go back to the beginning of the form for now.